docs/pages/functionality/filter-management.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Filter Management
+permalink: functionality/filter-management
+type: functionality
+language: en
+list: include
+---
+
+### What is an Envoy ?
+
+An <strong>Envoy</strong> is a high performance, programmable L3/L4 and L7 proxy that many mesh services implement like istio. There are network filters at the core of the Envoy’s connection and the traffic handling which when mixed with the filter chains allows the implementation of the higher-order functionalities for access control, transformation, data enrichment, auditing and many more. 
+
+### What is a WASM ?
+
+<strong>WASM (<i>Web Assembly</i>)</strong> is an extremely interesting way which greatly simplifies the process of extending Envoy with new capabilities. WASM is an efficient portable binary instruction format providing an embeddable and isolated execution environment.
```

Comment:
Invert the acronym and the long-form of WASM (first comes long-form). WebAssembly is one word.